src/aqua/chain/new_deals.aqua

```diff
@@ -45,7 +45,7 @@ func get_left_boundary(spell_id: SpellId, chain: ChainInfo) -> ?string:
             left <<- last_seen
     <- left
 
-func join_deals(spell_id: SpellId, logs: []DealCreated, left: string):
+func join_deals(spell_id: SpellId, logs: []DealMatched, left: string):
     Spell spell_id
 
     -- TODO: I assume that `logs` are sorted by `block_number`. Is that a correct assumption?
```

Comment:
still need that TODO?

Reply:
I'm still not convinced, haven't found it in specs.